regression/cbmc-java/phi-merge_uninitialized_values/PhiMergeUninitialized.java

```diff
@@ -0,0 +1,56 @@
+public class PhiMergeUninitialized {
+
+    public int dynamicAllocationUninitialized(Boolean trigger) {
+
+        Ephemeral obj;
+        if (trigger) {
+            obj = new Ephemeral(42);
+        } else {
+            obj = new Aetherial(20);
+        }
+
+        assert obj.val == 20;
+        return obj.val;
+    }
+
+    private Ephemeral local;
```

Comment:
This isn't really a local -- in Java we can't really have a local object at all (but we can have a local reference or pointer). Really the one labelled `dynamic` is using a local variable to store the pointer, and this one is using a member field to store it.